src/lib/controllers/tokenize-controller.js

```diff
@@ -145,4 +147,17 @@ export default class TokenizeController {
 
     return nextIdWordParts.join('-')
   }
+
+  static getNextTokenIdWordChangesType ({ tokenIdWord, lastTokenWordId, changeType, indexWord }) {
+    if (changeType === TokensEditController.changeType.SPLIT) {
+      return `${tokenIdWord}-s${indexWord}`
+    }
+    if (changeType === TokensEditController.changeType.MERGE) {
+      return `${tokenIdWord}-m${indexWord}`
+    }
+    if (changeType === TokensEditController.changeType.UPDATE) {
+      return `${tokenIdWord}-e`

```

Comment:
why no indexWord increment here? there could be multiple edits of a token, and each one technically changes it from what it was before, making it a new token needing a new id

Reply:
May be I didn't understand your idea here?
There was no examples for this case :)

I do wthis similiar you described for others:
<prev id> + -e

so it would look like this
Was `1-0-0` -> became `1-0-0-e` -> `1-0-0-e`
I didn't add here index, because there is only one word that is used in index

If you see here any other workflow, please describe it.

Reply:
Oh, I see in the changed comment. Will change.
Am I right that if in the following case <increment> would start again?

`1-0-0-e-2-m-2-e-1`